.github/workflows/checks.yml

```diff
@@ -4,6 +4,23 @@ name: Checks
 on: [push, pull_request]
 
 jobs:
+  swiftformat:
+    name: SwiftFormat
+    runs-on: macOS-latest
+    steps:
+      - uses: actions/checkout@v1
+      - name: Select Xcode 11.2.1
+        run: sudo xcode-select -switch /Applications/Xcode_11.2.1.app
+      - uses: actions/setup-ruby@v1
+        with:
+          ruby-version: '2.x'
+      - name: Install Bundler 2.0.2
+        run: gem install bundler --version 2.0.2
+      - name: Install Bundler dependencies
+        run: bundle install
+      - name: Swift format check
+        run: |
+          bundle exec rake style_correct
```

Comment:
@pepibumur is the given rake task `style_correct` intended for local usage only?

Reply:
Right. That task is intended to be used locally to autocorrect styles using SwiftLint & Swiftformat. Depending on how far you want to go with the task you can simply add a Rake task that runs Swiftformt, or implement a custom action that runs Swiftformat and use the GitHub annotations API to annotate the lines that have errors. Similarly to how SwiftLint does it. In case of the latter, you'll get a lot familiar with GitHub Actions. Up to you :)